source/navitiacommon/navitiacommon/utils.py

```diff
@@ -31,6 +31,11 @@
 import os
 import glob
 
+street_source_types = ['BANO', 'OSM']
+address_source_types = ['BANO', 'OSM', 'OpenAddresses']
+stop_area_source_types = ['FUSIO', 'BANO']
```

Comment:
Since for now we will always get the stop_area from kraken I don't think we need to configure anything about them